src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -209,10 +210,41 @@ public class HelixWatch3DViewModel : DefaultWatch3DViewModel
         private readonly Dictionary<string, List<Tuple<string, Vector3>>> labelPlaces
             = new Dictionary<string, List<Tuple<string, Vector3>>>();
 
+        //this code is grabbed from the helix source
+        //https://github.com/helix-toolkit/helix-toolkit/blob/develop/Source/HelixToolkit.SharpDX.Shared/Utilities/NVOptimusEnabler.cs#L15
+        //as of 2.24.0 this class is not compiled in their netcore targets.
+        /// <summary>
+        /// Enable dedicated graphics card for rendering. https://stackoverflow.com/questions/17270429/forcing-hardware-accelerated-rendering
+        /// </summary>
+        internal sealed class DYNNVOptimusEnabler
+        {
+            static DYNNVOptimusEnabler()
+            {
+                try
+                {
+
+                    if (Environment.Is64BitProcess)
+                        NativeMethods.LoadNvApi64();
+                    else
+                        NativeMethods.LoadNvApi32();
```

Comment:
so we have 32 bit support here just because this code has been copy pasted right >?

Reply:
yeah, I also asked about this class being removed here:
https://github.com/helix-toolkit/helix-toolkit/issues/2004 ... so - not sure it will work.

you can give this a read if you are interested - though it seems we might be able to do something now that .net supports native external callers ...

anyway, that blog post and this method don't seem to work the same way so there must be more to it than an exported member in the pe file. I guess?

For the time being, I think this is safe enough even if it does not work - we'll just have to pay attention to reports of users needing to explicitly set hardware accel again for sandbox - like back in the dark ages 😉 )